packages/react-server/src/render/render.tsx

```diff
@@ -77,14 +78,14 @@ export function createRender(render: RenderFunction, options: Options = {}) {
     const hydrationManager = new HydrationManager();
 
     function Providers({children}: {children: React.ReactElement<any>}) {
-      const [, Serialize] = useSerialized<Data>('x-quilt-data');
+      const [, Serialize] = useSerialized<Data>('quilt-data');
 
       return (
         <AsyncAssetContext.Provider value={asyncAssetManager}>
           <HydrationContext.Provider value={hydrationManager}>
             <NetworkContext.Provider value={networkManager}>
               {children}
-              <Serialize data={() => ctx.headers['x-quilt-data']} />
+              <Serialize data={() => ctx.state.quiltData} />
```

Comment:
I don't see a changelog update for this so I presume it doesn't need one. No user impact so makes sense.